src/app/pages/SearchResultsPage/GlobalSearchResultsView.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { getFilterForScope, getKeyForScope, getNameForScope } from '../../../types/searchScope'
+import { RouteUtils } from '../../utils/route-utils'
+import { SearchResults } from './hooks'
+import { NoResults } from './NoResults'
+import { SearchResultsFiltered } from './SearchResultsFiltered'
+import { useGetRosePrice } from '../../../coin-gecko/api'
+
+export const GlobalSearchResultsView: FC<{ searchResults: SearchResults }> = ({ searchResults }) => {
+  const { t } = useTranslation()
+  const roseFiatValue = useGetRosePrice().data
```

Comment:
`View` components should try to be for displaying data. I'd pass this down from a `Page` component

Reply:
Yes, it's like that after the latest rebase. (The page component gets the prices)